### ISSUE TITLE
swagger-ui operations-sorter function not working

### ISSUE THREAD

**Issue by fxrobin** [quarkusio/quarkus#15642]
**Describe the bug**
the javascript function defined in the property `quarkus.swagger-ui.operations-sorter` is not working.

**Expected behavior**
When defined, the JavaScript function should be called.

**Actual behavior**
Only "alpha" and "method" parameters are working.

**Configuration**
```properties
quarkus.swagger-ui.operations-sorter=function (a, b) { var order = { "get": "0", "post": "1", "put": "2", "delete": "3" }; return order[a.method].localeCompare(order[b.method]); }
```

**Environment (please complete the following information):**
 - Quarkus version : 1.10.3 to 1.12.1 (and maybe before)

**Possible cause**

Single quotes are generated around the the property "operationsSorter" : 

```
window.onload = function() {
                const ui = SwaggerUIBundle({
                url: '/v1/openapi',
                    dom_id: '#swagger-ui',
                    deepLinking: true,
                    operationsSorter: 'function (a, b) { var order = { "get": "0", "post": "1", "put": "2", "delete": "3" }; return order[a.method].localeCompare(order[b.method]); }',
                    validatorUrl: 'none',
                    presets: [SwaggerUIBundle.presets.apis,SwaggerUIStandalonePreset],
                    plugins: [SwaggerUIBundle.plugins.DownloadUrl],
                    layout: 'StandaloneLayout',
                })
```

See https://stackoverflow.com/questions/24951268/sort-api-methods-in-swagger-ui

**Comment by phillip-kruger** [comment]
Thanks, I'll fix this a.s.a.p

**Comment by phillip-kruger** [comment]
This will be fixed here: https://github.com/smallrye/smallrye-open-api/pull/742

Once SmallRye is released and pulled into Quarkus, I'll close this issue.

**Comment by fxrobin** [comment]
> This will be fixed here: [smallrye/smallrye-open-api#742](https://github.com/smallrye/smallrye-open-api/pull/742)
> 
> Once SmallRye is released and pulled into Quarkus, I'll close this issue.

Thanx, that was a fast fix!

**Comment by JBartscher** [comment]
How could I escape this if I use an application.yaml instead of an application.properties?
